pallets/runtime/src/update_did_signed_extension.rs

```diff
@@ -0,0 +1,204 @@
+use crate::{runtime, Runtime};
```

Comment:
This file is no longer needed.

Reply:
hmm, It should get deleted when I resolve the conflicts. Anyway I will delete this